.devcontainer/devcontainer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "image": "mcr.microsoft.com/azterraform:latest",
+  "image": "ubuntu",

```

Comment:
intentionally?

Reply:
Yeah I could not get az cli working to test the deployment in Azure. So I switched to ubuntu